src/promptflow/promptflow/_sdk/_configuration.py

```diff
@@ -195,10 +198,15 @@ def get_or_set_installation_id(self):
         user_id = self.get_config(key=self.INSTALLATION_ID)
```

Comment:
let's rename all user_Id in this function to installation_id

Reply:
ok